### PR TITLE
feat(embed): Implement template loading from embedded filesystem

### PR DESCRIPTION
**Pull request by bliepp**
This pull requests implements a function `AddFromFS` as requested by #30 for everything matching the interface `fs.FS` from the standard library (primarily `embed.FS` of course) to get rid of ugly workarounds.

*This PR breaks compatibility with go version 1.15 and below as it relies on the `fs.FS` interface*

### CODE REVIEW

**Comment by bambamboole** [comment]
Can I somehow support, that we get this merged? 
My current project would benefit from this :)

**Comment by bliepp** [comment]
> Can I somehow support, that we get this merged? 
> My current project would benefit from this :) 

From my perspective it seems ready to merge.

**Comment by bliepp** [comment]
> Can I somehow support, that we get this merged? 
> My current project would benefit from this :) 

This repo doesn't seem to have significant activities for the last 4 months so I guess it's just a matter of time until the owners merge it. From a technical perspective it should be ready to merge.

**Comment by mrpineapples** [comment]
Would love to get this merged in 🙏🏼

File: renderer.go
Comment (mrpineapples) [inline renderer.go:19]: Would be nice to also have an `AddFromFSFuncs` if possible.

Comment (bliepp) [reply renderer.go:19]: It's been a while since I created this PR and the maintainers don't seem to care for a merge. So I'll wait for it to be merged before putting any more effort into this.

Comment (mrpineapples) [reply renderer.go:19]: Yea I see, that's a shame. But I used the workaround in #30, not the worst thing in the world 😭

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
https://github.com/gin-contrib/multitemplate/releases/tag/v1.0.0

**Pull request merged**
